docker/prod/Dockerfile

```diff
@@ -187,10 +192,6 @@ RUN cp Gemfile.lock.bak Gemfile.lock && rm Gemfile.lock.bak
 
 RUN ./docker/prod/setup/postinstall.sh
 
-# We need this so puma is allowed to create the tmp/pids folder and files
```

Comment:
This was actually meant to happen for the slim, not the AIO container. The latter doesn't need it.